exportReport/Export/ReportExport.java

```diff
@@ -0,0 +1,325 @@
+			transformer.transform(source, result);
+		}
+		catch (TransformerConfigurationException | TransformerFactoryConfigurationError e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: Transformer Configuration problem, see log for more details.", new Object[] { absolutePath }));
+		}
+		catch (TransformerException e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: Transformer problem, see log for more details.", new Object[] { absolutePath }));
+		}
+		return f_result;
+	}
+	
+	public static File JSON_EXPORT(ViewDelegate view, Logger logger, String absolutePath, String p_result, String p_source) 
+	{
+		int PRETTY_PRINT_INDENT_FACTOR = 4;
+		
+		File fXmlFile = new File(p_source);
+		DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+		DocumentBuilder dBuilder = null;
+		try
+		{
+			dBuilder = dbFactory.newDocumentBuilder();
+		}
+		catch (ParserConfigurationException e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: Parser Configuration Exception, see log for more details.", new Object[] { absolutePath }));
+		}
+		Document doc = null;
+		try
+		{
+			doc = dBuilder.parse(fXmlFile);
+		}
+		catch (SAXException e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: SAX Exception, see log for more details.", new Object[] { absolutePath }));
+		}
+		catch (IOException e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: IO Exception, see log for more details.", new Object[] { absolutePath }));
+		}
+		String temp = getStringFromDoc(doc);
+		String jsonPrettyPrintString = null;
+		try
+		{
+			JSONObject xmlJSONObj = XML.toJSONObject(temp);
+			jsonPrettyPrintString = xmlJSONObj.toString(PRETTY_PRINT_INDENT_FACTOR);
+		}
+		catch (JSONException e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: JSON Exception, see log for more details.", new Object[] { absolutePath }));
+		}
+		File f = null;
+		try
+		{
+			f = write(p_result, jsonPrettyPrintString, false);
+		}
+		catch (Exception e)
+		{
+			logger.error(e.getMessage(), e);
+			view.showMessageDialog(MessageFormat.format("Error: Exception, see log for more details.", new Object[] { absolutePath }));
+		}
+		return f;
+	}
+	
+	public static String getStringFromDoc(Document doc)
+	{
+		DOMImplementationLS domImplementation = (DOMImplementationLS) doc.getImplementation();
+		LSSerializer lsSerializer = domImplementation.createLSSerializer();
+		return lsSerializer.writeToString(doc);
+	}
+
+	private static File write(String path, String str, Boolean append) throws Exception
+	{
+		File f = new File(path);
+
+		try (Writer writer = Channels.newWriter(new FileOutputStream(f.getAbsoluteFile(), append).getChannel(), "UTF-8"))
```

Comment:
It's likely handled but this string `UTF-8` appears in numerous locations, for consistency and ease of maintenance it should probably be a constant.